config/roguelike_dungeons/settings/loot_tools.json

```diff
@@ -0,0 +1,141 @@
+{
+	"name": "loot_tools",
+	"loot_rules": [
+	{"level": [0, 1, 2, 3, 4], "type": "TOOLS", "each": true, "quantity": 2, "loot": [
+		{"data": {"name": "berriespp:itemSpade", "meta": 0}, "weight": 1},
+		{"data": {"name": "berriespp:itemlens", "meta": 0}, "weight": 1},
+		{"data": {"name": "minecraft:flint_and_steel", "meta": 0}, "weight": 1},
+		{"data": {"name": "minecraft:fishing_rod", "meta": 0}, "weight": 1},
+		{"data": {"name": "minecraft:shears", "meta": 0}, "weight": 1},
+		{"data": {"name": "minecraft:lead", "meta": 0}, "weight": 1},
+		{"data": {"name": "minecraft:clock", "meta": 0}, "weight": 1},
+		{"data": {"name": "minecraft:bucket", "meta": 0}, "weight": 1},
+		{"data": {"name": "IguanaTweaksTConstruct:clayBucketFired", "meta": 0}, "weight": 1},
+		{"data": {"name": "IC2:itemTreetap", "meta": 0}, "weight": 1},
+		{"data": {"name": "SpiceOfLife:lunchbox", "meta": 0}, "weight": 1},
+		{"data": {"name": "SpiceOfLife:lunchbag", "meta": 0}, "weight": 1},
+		{"data": {"name": "harvestcraft:mixingbowlItem", "meta": 0}, "weight": 1},
+		{"data": {"name": "harvestcraft:potItem", "meta": 0}, "weight": 1},
+		{"data": {"name": "harvestcraft:saucepanItem", "meta": 0}, "weight": 1},
+		{"data": {"name": "harvestcraft:skillitItem", "meta": 0}, "weight": 1},
+		{"data": {"name": "Forestry:scoop", "meta": 0}, "weight": 1},
+		{"data": {"name": "BiomesOPlenty:scytheDiamond", "meta": 0}, "weight": 1},
+		{"data": {"name": "ifu:ifu_buildingKit", "meta": 10}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 12}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 10}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 18}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 16}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 20}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 36}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 22}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 34}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 14}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 44}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 26}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 46}, "weight": 1},
+		{"data": {"name": "gregtech:gt.metatool.01", "meta": 24}, "weight": 1},
```

Comment:
I think giving out GT tools is a good idea, but those and many other items need to have NBT data specified in order to be usable. Use the ingame command `/nbtedit` on a tile entity like a chest or `/nbtedit item` on a hand held item to view and copy the required NBT data to replicate the item of your choice.

![image](https://github.com/user-attachments/assets/d5b7a006-80a9-44c3-a907-eedcb950f27a)

I realized this when I came across an invalid 0/0 hammer.
![invalid-hammer](https://github.com/user-attachments/assets/3b09f53d-13f6-4625-9eca-d40f9e228f17)